src/window/win.cc

```diff
@@ -993,6 +993,14 @@ namespace SSC {
                           return S_OK;
                         }
 
+                        if (msg.name == "stdout") {
```

Comment:
why add this here instead of changing it in `bridge.cc` to handle windows?

Reply:
Resolved, works via stdWrite now